newrelic_lambda_cli/integrations.py

```diff
@@ -342,9 +400,24 @@ def update_log_ingestion_function(input):
 
         # We can't change props during import, so let's set them to their current values
         lambda_client = input.session.client("lambda")
-        old_props = lambda_client.get_function_configuration(
-            FunctionName="newrelic-log-ingestion"
+        unique_log_ingestion_name = get_unique_newrelic_log_ingestion_name(
```

Comment:
We probably need to pass this name (unique_log_ingestion_name) into the `_import_log_ingestion_function` since here the stack id exists and by the time it calls `_import_log_ingestion_function` it doesn't.